src/modm/platform/core/sam/module.lb

```diff
@@ -35,7 +35,8 @@ def build(env):
     # delay code that must be tuned for each family
     # (cycles per loop, setup cost in loops, us shift)
     tuning = {
-        "d": (3,  4, 5), # CM0 tested on D21 in RAM
+        "d": (3, 4, 5), # CM0 tested on D21 in RAM
+        "g": (6, 4, 5) # G55
```

Comment:
6 cycles per loop? that would be quite an outlier…

Reply:
Indeed. Empirically, that's what works. I suspect the loop is not getting into RAM and there's something else I need to do there, and/or I need to enable some caching. It's on my agenda to dig more.

Reply:
I wrote a blog post about this, in case you want more info: https://blog.salkinium.com/modm-delay/

Reply:
I confirmed delay_ns is located in RAM. One possible explanation is that my clock is half what I think it is, but delay_us appears to be using the DWT and that also yields the correct delay, so I think that disproves the wrong clock theory. 

There is a tightly coupled memory, which is apparently higher performance than the other RAM. Perhaps fastdata and fastcode should go into this region, but I'd like to kick that can down the road to another PR.